client/state/sites/schema.js

```diff
@@ -57,7 +57,8 @@ export const sitesSchema = {
 						is_free: { type: 'boolean' }
 					}
 				},
-				single_user_site: { type: 'boolean' }
+				single_user_site: { type: 'boolean' },
+				updates: { type: 'object' },
```

Comment:
Tested this, and looks like it's ok here with `undefined`. Just be careful with some of our endpoints, at times they may return different types or `null` for optional properties.